public/templates/footer-template.html

```diff
@@ -5,7 +5,7 @@
     * {
       font-family: 'Red Hat Text';
       font-style: italic;
-      font-size: 9px;
+      font-size: 8px;
```

Comment:
I know the value might not make sense at first glance, however, these values were taken from UX designs. You can create a custom footer if you require this change.

Reply:
Aha, but having 8px font will cause the main content to overlap over the footer everywhere. Do we still want to keep 9px as default?

Reply:
AFAIK It does not overlap in other reports. You might want to change the bottom margin of your template. There is a `browserMargins` config available when registering your template to the template mapper.

Reply:
Yeah, initially I opted for changing the margin. But it did not give any effect. Anyways, if we see such a problem in the future, we know the solution. I will create a custom footer.